README.md

```diff
@@ -43,7 +75,89 @@ A list of available providers can be found [here](https://openfeature.dev/docs/r
 
 For complete documentation, visit: https://openfeature.dev/docs/category/concepts
 
-### Hooks
+### Context-aware evaluation:
+
+Sometimes the value of a flag must take into account some dynamic criteria about the application or user, such as the user location, IP, email address, or the location of the server.
+In OpenFeature, we refer to this as [`targeting`](https://openfeature.dev/specification/glossary#targeting).
+If the flag system you're using supports targeting, you can provide the input data using the `EvaluationContext`.
+
+```go
+// add a value to the global context
+openfeature.SetEvaluationContext(openfeature.NewEvaluationContext(
+    "",
```

Comment:
minor - may be it's best to add value to targetingKey

Ex: `	openfeature.SetEvaluationContext(openfeature.NewEvaluationContext("foo", map[string]interface{}{"myGlobalKey": "myGlobalValue"}))`